src/middlewared/middlewared/plugins/directoryservices.py

```diff
@@ -3,14 +3,21 @@
 import os
 import struct
 import tdb
+import pickle
+import pwd
+import grp
 
 from base64 import b64encode, b64decode
 from middlewared.schema import accepts
 from middlewared.service import Service, private
 from middlewared.plugins.smb import SMBCmd, SMBPath
-from middlewared.utils import run
+from middlewared.utils import filter_list, osc, run
 from samba.dcerpc.messaging import MSG_WINBIND_OFFLINE, MSG_WINBIND_ONLINE
 
+OS_TYPE_FREEBSD = 0x01
+OS_TYPE_LINUX = 0x02
+OS_FLAG = int(osc.IS_FREEBSD) + (int(osc.IS_LINUX) << 1)
```

Comment:
This is like C in Python which doesnt make the most sense as we are not hurting for saving a few bytes.
Mostly just noting the fact, no need to change :P

Reply:
That's one WEIRD line of code